### ISSUE TITLE
Loss of Navigation in Drum Kit Options with left arrow

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#32799]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

If you open the Options menu for any instrument on the drum pad you can navigate through the two options of Define Keyboard shortcut and Delete Pad by using Up/Down arrows or even right arrow.  But if you press a left arrow you seem to be back out of this options area and on the list of instrument options.

STEPS:
1.  Create a score with a drum kit, and navigate to that staff.  Press tab several times till you get to the Percussion area and continue till you hear an instrument followed by "Options", such as "Bass Drum 1 Options not selected".  Press Enter.
2. Press Up or Down arrows several times and see you can cycle through what seem to be the only two options in this area, "Define Shortcut" and "Delete Pad".
3. Press right arrow a few times and notice the same result as above.
4. Now Press left arrow and note you are back on the instrument options list.  At this point no combo of tab or arrows seems to get you back to where you were.
Note also the word "not selected" is spoken after every instrument options name.  Here, like in other places this seems unnecessary as other than pressing enter to open it what else can be done, if they can't actually be selected as such, or are automatically selected with navigation then this is not needed.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.7 beta

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by avvvvve** [comment]
If you press Shift + Tab when on an options button, it will return navigation to the main body of the drum pads.
